pisa/analysis/analysis.py

```diff
@@ -808,8 +812,10 @@ def fit_hypo_inner(self, data_dist, hypo_maker, metric, minimizer_settings,
         fit_info = OrderedDict()
         fit_info['metric'] = metric
         fit_info['metric_val'] = metric_val
-        if blind:
-            hypo_maker.reset_free()
+        if blind > 1:  # only at stricter blindness level
```

Comment:
just out of curiosity: why is this only needed during blind fits but not otherwise?

Reply:
At the strictest blindness level, no information is stored about the fits. This reset command overwrites any fitted values to make sure they can't be stored even accidentally later.

Reply:
Aha, ok got it! Sounds good. I'll merge this PR then